packages/perseus/src/perseus-types.ts

```diff
@@ -909,6 +909,125 @@ export type PerseusGraphTypeRay = {
     startCoords?: CollinearTuple;
 } & PerseusGraphTypeCommon;
 
+type AngleGraphCorrect = {
```

Comment:
Not sure if all these types and the object/arrays I add here should be in this file or another one. Was considering the types file in Interactive Graph, though there might be a better place for that.

Perhaps the object/arrays should just go directly in a test data file since it was mainly to create rubrics in the validator test? Wasn't sure if they might be helpful in the future.

Reply:
I think it's fine to put them here. It's an easy decision to change later, in any case.

Reply:
`perseus-types.ts` is for external facing types. I think it's correct to have them here since the rubric will be an external data structure now.